event/event.go

```diff
@@ -256,7 +256,10 @@ func (r *MessageSubscription) Serve(handler MessageHandler) error {
 					const size = 64 << 10
 					buf := make([]byte, size)
 					buf = buf[:runtime.Stack(buf, false)]
-					slog.Error("panic: message subscription: handling message", "message", msg, "stack_trace", string(buf))
```

Comment:
Not getting why the old error was producing: 

"!ERROR:json: unsupported type: func(func(inte"

msg is a struct...and stack_trace is a big string... :confused:

Reply:
Ah it is related to "message" being used by google cloud logging as a special field